components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div>
+        <span class="mb-3.5 text-center text-zinc-900 text-2xl font-bold font-['Noto Sans JP'] leading-normal">
+            {{ $t('modSubmissionForm.healthcareProfessionalId') }}
+        </span>
+        <div class="flex flex-col mt-4">
+            <div class="justify-start items-start flex">
+                <input 
+                    type='text'
+                    placeholder='Search by name or ID to add a doctor'
+                    class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400 text-neutral-600 text-sm font-normal font-['Noto Sans JP'] placeholder-gray-300"
+                />
+                <SVGLookingGlass role="img" title="searching icon" class="relative right-8 top-3 w-6 h-6"/>
```

Comment:
does this need to be relative?

Reply:
In the Figma the image is in the search box, and this was the only way to put an image in a  input field. 
I can also use a div to create a search bar and than I don't have to use the relative, which one do you recommend?

Reply:
I think the container div makes more sense. Relative is also ok, but it inherently comes with more risk of UI abnormalities since it renders dependent on another element.

Reply:
Ah I know why I went with input, because of the placeholder text. I think that requires some css/tailwind skill. I will try if I can figure it out.